pkg/build/webhook/controller.go

```diff
@@ -71,7 +71,11 @@ func (c *controller) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		notFound(w, "Plugin ", uv.plugin, " not found")
 		return
 	}
-	revision, proceed, err := plugin.Extract(buildCfg, uv.secret, uv.path, req)
+	revision, evptr, proceed, err := plugin.Extract(buildCfg, uv.secret, uv.path, req)
+	envvars := []kapi.EnvVar{}
+	if evptr != nil {
+		envvars = *evptr
+	}
```

Comment:
seems like you can simplify this and the other spot where you do the same thing if you just make Extract return `[]kapi.EnvVar` instead of `*[]kapi.EnvVar`, was there a reason to have it return the pointer?